doc/system_specific_setup.rst

```diff
@@ -137,4 +181,8 @@ You can now install the extensions
     > sudo pip install pytest-flakes
     > sudo pip install pytest-pep257
 
+If you don't have root access then you can specify the ``--user`` argument to
+install packages in a user-local directory -- see the instructions on
+:ref:`install_fparser` above.
```

Comment:
for consistency also show how to uninstall. I know this should have been there before.